test/utility-test.js

```diff
@@ -99,145 +99,168 @@ describe('Rawcrawl Util', function() {
       expect(obj).to.be.an('array');
     });
   });
-  describe('#getRowById()', function() {
-    it("Should throw an error when given invalid id", function(done) {
-      var id = -1;
-      var logsql = false;
-      rc_util.getRowById(db_url, id, logsql)
-      .catch(function(error) {
-        assert.strictEqual(error, "Invalid id range");
-      }).then(done, done);
-    });
-    it("Shouldn't throw an error when given valid id", function() {
-      var id = 1;
-      var logsql = false;
-      rc_util.getRowById(db_url, id, logsql);
-    });
-    it("Should return an object with expected properties", function(done) {
-      this.timeout(10000);
-      var id = 1;
-      var logsql = false;
-      rc_util.getRowById(db_url, id, logsql)
-      .then(function(row) {
-        if (row) {
-          expect(row).to.have.property('id');
-          expect(parseInt(row.id)).to.be.a('number');
-
-          expect(row).to.have.property('start_at');
-          expect(row.start_at).to.be.a('date');
-
-          expect(row).to.have.property('end_at');
-          expect(row.end_at).to.be.a('date');
-
-          expect(row).to.have.property('entry_ipp');
-          expect(row.entry_ipp).to.be.a('string');
-
-          expect(row).to.have.property('data');
-          expect(row.data).to.be.a('string');
-
-          expect(row).to.have.property('exceptions');
-          expect(row.exceptions).to.be.a('string');
-
-          expect(row).to.have.property('created_at');
-          expect(row.created_at).to.be.a('date');
-
-          expect(row).to.have.property('updated_at');
-          expect(row.updated_at).to.be.a('date');
-        }
-      })
-      .then(done, done);
+});
+
+/* Post two crawls to db for testing */
+var Crawler = require('../src/lib/crawler.js').Crawler;
+var _ = require('lodash');
+var src = require('../src/program');
+var Promise = require('bluebird');
+var db_url = 'postgres://postgres:postgres@127.0.0.1:5432/circle_test';
+
+var maxRequests = 100;
+var ipp = '162.217.98.90:51235';
+var crawler = new Crawler(maxRequests);
+crawler.getCrawl(ipp).then(function(response) {
+  src.store(response, db_url, false).then(function() {
+    src.store(response, db_url, false).then(function() {
+      testDB();
     });
```

Comment:
Put all of this into a `before()` or `beforeEach()`, then you won't have to put all the tests into a function